### DISCUSSION TITLE
I would like to have my own desktop environment but using Linux Mint codebase

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#548] [Development]
[Metropicture presentation](https://github.com/metropicture/help/blob/master/presentation.md)

I basically just want the Windows 8.1 user interface (desktop and default big picture mode)

![image](https://github.com/user-attachments/assets/99cdff26-7a3f-4835-b992-5d77705d7c6b)

(Short video for side by side gestures)

[![image](https://github.com/user-attachments/assets/8a18e160-94d9-4e1c-b9a7-c0653620d686)](https://www.youtube.com/watch?v=SMz_rd92pyQ)

I would like to know if forking Cinnamon is enough. I know that forking a theme only allows you to change CSS; not some programming and layout changes.

**Comment by billyswong** [comment]
I think you should take a look at some existing projects such as https://ubuntusway.com/ and https://swaywm.org/
